src/simulator/aircrafts/F16/aerodynamics.jl

```diff
@@ -0,0 +1,321 @@
+using FlightMechanics
+using FlightMechanics.Simulator.Models
+import FlightMechanics.Simulator.Models: calculate_aerodynamics
+
+export F16Aerodynamics, calculate_aerodynamics
+
+
+# TODO: CDα, CLα, CYβ... could be included here
+struct F16Aerodynamics<:Aerodynamics
+    wind_pfm::PointForcesMoments
+    wind_coeff_pfm::PointForcesMoments
+    body_pfm::PointForcesMoments
+    body_coeff_pfm::PointForcesMoments
+end
+
+
+α_data = [-10. -5. 0. 5. 10. 15. 20. 25. 30. 35. 40. 45.]  # deg
+β_data = [0. 5. 10. 15. 20. 25. 30.]  # deg
+
+de_data = [-24. -12.   0.  12.  24.]  # deg
+
+# TODO: move this constants to aircraft parameters
```

Comment:
They cannot be moved to aircraft becase an aircraft instance would be necesary to access them. They are left here and used from FCS (where they where coded again).

This could be accomplished once the aircraft struct has been refactored